netmesh/model/netmesh/netmesh.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package netmesh;
+
+enum ApiVersion {
+    NETMESH_API_VERSION_1 = 0;
```

Comment:
Do we want to use an integer representation here? I would prefer to see some form of semver for api versions. It's a bit more noisy, but I think it would be worth it.

Reply:
A quick search leads me to this:

https://github.com/blang/semver

I think we can pull that in, and remove versions from the protobuf here completely, making it a part of the initial handshake, as you mention below.